tests/app/main/test_frameworks.py

```diff
@@ -3776,6 +3776,38 @@ def test_should_be_a_503_if_email_fails(self, send_email, data_api_client):
 @mock.patch('app.main.views.frameworks.data_api_client', autospec=True)
 @mock.patch('app.main.views.frameworks.count_unanswered_questions')
 class TestG7ServicesList(BaseApplicationTest):
+    def _assert_incomplete_application_banner_not_visible(self, doc):
+        assert not doc.xpath('//*[@class="banner-information-without-action"]')
```

Comment:
"Xpath doesn't exist" is quite a brittle test, easy to get false positives (e.g. the mark up changes, the banner shows up when it shouldn't, but this tasty still says everything is fine). Might be good to have a slightly "blunter" check for the banner heading text anywhere in the page.

Reply:
Fair point, will update